SR2019/2019-05-28-single-item-kit-chase.md

```diff
@@ -0,0 +1,27 @@
+---
+to: Teams with outstanding single items of kit
+subject: Outstanding kit
+---
+
+Hello,
+
+According to our records, you still have some items from the robot kit you borrowed during the SR2019 competition. As the competition has ended, we would like this back!
+
+At the end of the competition, we checked your kit in, and some items were missing. Below are the items we require back:
+
+## Missing items
+
+-
+-
+
+Please search anywhere you may have worked on your robot for these items. If you are unable to find any of these items, please [let us know](mailto:teams@studentrobotics.org).
+
+## Shipping items
+
+Please insure the kit during shipping and use a courier which supports tracking, to ensure the kit isn't lost.
+
+Please securely pack the kit and ship it to:
+
+1 Alie Street, Whitechapel, London, E1 8DE
```

Comment:
Just to check -- are you going to add my name & company name to this before sending the email? (I appreciate them not being in git though).

Reply:
I was wondering this. I wasn't intending to, but I definitely can. I highly suspect that's going to make it easier on your end to receive the items?

Reply:
Yeah, it's pretty essential that it's got more than just the building address on the parcel. There's several companies in the building, so reception wouldn't be best pleased if they got a bunch of partially addressed parcels.